dynamics-nav/SETFILTER-Function--TestPage-Filter-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETFILTER Function (TestPage Filter)"
+description: SETFILTER Function (TestPage Filter) Applies a filter to the specified field on a test page. This topic contains information on the SETFILTER Function (TestPage Filter) syntax and parameters.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.